### PR TITLE
Add kinesis stream for logs ingestion

### PR DESCRIPTION
**Pull request by TBonnin**
In addition accept tcp input, Logstash is now setup to read logs from a Kinesis stream.

### CODE REVIEW

File: config/logstash-indexer.conf [hunk 1]
```diff
@@ -3,6 +3,13 @@ input {
         port  => 6379
         codec => json_lines
     }
+
+    kinesis {
+        kinesis_stream_name => '@@KINESIS_STREAM_NAME'
+        application_name => '@@KINESIS_APPLICATION_NAME'
+        region => '@@KINESIS_REGION'
+        codec => json { }
+    }
```
Comment (TBonnin) [inline config/logstash-indexer.conf]: I kept both inputs (tcp and kinesis). I am happy to discuss though

Comment (satterly) [inline config/logstash-indexer.conf]: Sounds fine to me. Gives people the option to use one or the other, right?

**Comment by satterly** [comment]
👍 Thanks!

File: config/logstash-indexer.conf (hunk 1 above)
Comment (TBonnin) [inline config/logstash-indexer.conf]: or both 😁

**Comment by TBonnin** [comment]
Thank you @satterly

**Pull request merged**